config_defaults.yml

```diff
@@ -9,7 +9,14 @@ Gemfile:
   - gem: rspec-puppet-facts
     version: '>= 1.7'
   - gem: puppetlabs_spec_helper
-    version: '>= 0.8.0'
+    version: ">= 1.1.0', '<= 2.1.0"
+  - gem: parallel_tests
+    options:
+      platforms:
```

Comment:
> So this part fails on older rubies. I haven't seen an option to do this on all buy ruby_19 platforms. Is there any?

You'd need to move it to the Gemfile proper, using `gem 'parallel_tests', [...] if RUBY_VERSION > '1.9'`.